### PR TITLE
fix(create-local-vue): fix max range stack size (fix #1768)

### LINKED ISSUE #1768: Infinite loop on uncaught error when using createLocalVue()
### Subject of the issue

There is an infinite loop in the vue-test-utils error handler when using `createLocalVue` more than once. Everything works correctly for the first test, but subsequent tests that use their own local vue instance will cause massive noise in the console
I have tested this with version 1.1.2

### Steps to reproduce

- Create a component with an uncaught error
- Mount the component using a distinct local vue in beforeEach (createLocalVue)
- Have more than one test

### Expected behaviour

Each test should run, and produce only the errors triggered by the code

### Actual behaviour

You should see an error similar to this in the console:
```
  console.error node_modules/vue/dist/vue.runtime.common.dev.js:621
    [Vue warn]: Error in config.errorHandler: "RangeError: Maximum call stack size exceeded"

  console.error node_modules/vue/dist/vue.runtime.common.dev.js:1884
    RangeError: Maximum call stack size exceeded
        at Array.push (<anonymous>)
        at getParent (...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2799:17)
        at findAllParentInstances (...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2805:3)
        at errorHandler (...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2912:32)
        at ...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2929:5
        at Array.forEach (<anonymous>)
        at errorHandler (...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2928:26)
        at ...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2929:5
        at Array.forEach (<anonymous>)
        at errorHandler (...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2928:26)
        at ...\node_modules\@vue\test-utils\dist\vue-test-utils.js:2929:5
       (...)
```

### Possible Solution

After debugging the code, it is apparent that the first call to createLocalVue() does not have an errorHandler set. Subsequent calls, however, have their error handler set to the one in vue-test-utils. Whenever the child component has an uncaught error, it will call the vue-test-utils handler, which will call the parent's (our local vue) handler. Since it is the same handler, with the same arguments, there is infinite recursion

There are many possible solutions, but a workaround is to simply pass an empty  errorHandler function:
```
createLocalVue({ errorHandler: () => {}})
```

### PR DESCRIPTION
**Pull request by antoinerey**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch.
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] All tests are passing: https://github.com/vuejs/vue-test-utils/blob/dev/.github/CONTRIBUTING.md#development-setup
- [x] New/updated tests are included

**Other information:**

- This PR fixes the issue #1768. 
- It looks like it's caused by Vue instance being polluted between tests (more details on the issue itself: https://github.com/vuejs/vue-test-utils/issues/1768#issuecomment-776926894).

### CODE REVIEW

**Comment by antoinerey** [comment]
While this PR is expected to fail, it seems that every checks are green which is quite confusing. 🤔 😁

File: test/specs/create-local-vue.spec.js [hunk 1]
```diff
@@ -232,3 +233,32 @@ describeWithShallowAndMount('createLocalVue', mountingMethod => {
...
 })
+
+// TODO: Clean this up, it's only useful while debugging the issue.
+describe.only('Test', () => {
+  // When setting the array length value to `1`, then the issue disappears.
+  // However, when setting it to anything bigger, the issue appears.
+  const sets = Array.from({ length: 2 })
+
+  it.each(sets)(
+    // Note, the message may not be the final one, based on what I discover.
```
Comment (lmiller1990) [inline test/specs/create-local-vue.spec.js]: can you put link to the issue? it will be difficult to remember what this test is really doing

**Comment by lmiller1990** [comment]
Let's hope this does not break anything :eyes:

**Comment by codfish** [comment]
@lmiller1990 what's the release schedule for the project like? When do you think we'll see this fix released into the wild?

**Comment by lmiller1990** [comment]
There is no set schedule, it's just whenever I do one. I can do a release this week. Maybe we should investigate sem-release and automate things...

If no release within a few days, ping me again, but I should be able to do it soon.

**Pull request merged**